contracts/treasury/LiquidationStrategyBase.sol

```diff
@@ -262,32 +266,33 @@ abstract contract LiquidationStrategy is ILiquidationStrategy, PermissionAdmin,
     emit RewardPoolSet(_pool);
   }
 
-  function _updateWhitelistedToken(address[] memory _tokens, bool _isAdd) internal {
-    for(uint256 i = 0; i < _tokens.length; i++) {
+  function _updateWhitelistedLiquidators(address[] memory _liquidators, bool _isAdd) internal {
+    for(uint256 i = 0; i < _liquidators.length; i++) {
       if (_isAdd) {
-        _whitelistedTokens.add(_tokens[i]);
+        _whitelistedLiquidators.add(_liquidators[i]);
       } else {
-        _whitelistedTokens.remove(_tokens[i]);
+        _whitelistedLiquidators.remove(_liquidators[i]);
       }
-      emit WhitelistedTokenUpdated(_tokens[i], _isAdd);
+      emit WhitelistedLiquidatorUpdated(_liquidators[i], _isAdd);
     }
   }
 
-  function _updateWhitelistedLiquidator(address _liquidator, bool _isAdd) internal {
-    if (_isAdd) {
-      _whitelistedLiquidators.add(_liquidator);
-    } else {
-      _whitelistedLiquidators.remove(_liquidator);
+  function _updateWhitelistedPriceOracles(address[] memory _oracles, bool _isAdd) internal {
+    for(uint256 i = 0; i < _oracles.length; i++) {
+      if (_isAdd) {
+        _whitelistedPriceOracles.add(_oracles[i]);
+      } else {
+        _whitelistedPriceOracles.remove(_oracles[i]);
+      }
+      emit WhitelistedPriceOracleUpdated(_oracles[i], _isAdd);
     }
-    emit WhitelistedLiquidatorUpdated(_liquidator, _isAdd);
   }
 
   function _setLiquidationSchedule(
     uint128 _startTime,
     uint64 _repeatedPeriod,
     uint64 _duration
   ) internal {
-    require(_repeatedPeriod > 0, 'repeatedPeriod == 0');
```

Comment:
because 0 repeated period means liquidations are disabled. but it makes more sense to go with ur suggestion of using 0 duration to disable liquidations instead.

Reply:
Change to use duration == 0 to disable the liquidation